chain/indexer/src/streamer/fetchers.rs

```diff
@@ -6,11 +6,11 @@ use actix::Addr;
 use futures::stream::StreamExt;
 use tracing::warn;
 
+use near_indexer_primitives::IndexerExecutionOutcomeWithOptionalReceipt;
 pub use near_primitives::hash::CryptoHash;
 pub use near_primitives::{types, views};
```

Comment:
Off-topic; we don't need to solve it now, yet I want to bring this to attention. Do we actually need them to be `pub`? It caught my eye that this API surface is a bit odd.

Reply:
I don't remember the reason so I've dropped `pub` from the places I believe it's not necessary